card/spec/models/card/cardtype_spec.rb

```diff
@@ -84,12 +84,12 @@ class Card
 end
 
 
-describe Card, "Normal card with dependents" do
+describe Card, "Normal card with descendants" do
   before do
     @a = Card['A']
   end
-  it "should confirm that it has dependents" do
-    expect(@a.dependents.length).to be > 0
+  it "should confirm that it has descendants" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.